docs/articles/infrastructure/ci-cd/yocto-cache.rst

```diff
@@ -0,0 +1,101 @@
+Yocto cache
+===========
```

Comment:
This looks like a how-to basically. I think the title and file name should reflect that something is to be done, e.g. set up a yocto cache, and there should be a label that indicates what type of article it is (see recent additions of labels).

Reply:
Fair point.